src/test/java/de/jtheb/fh/lagerverwaltung/MainTest.java

```diff
@@ -1,15 +1,30 @@
 package de.jtheb.fh.lagerverwaltung;
 
+import de.jtheb.fh.lagerverwaltung.entities.Compartment;
+import de.jtheb.fh.lagerverwaltung.entities.Shelf;
+import de.jtheb.fh.lagerverwaltung.entities.Warehouse;
+import de.jtheb.fh.lagerverwaltung.entities.Item;
 import org.junit.Test;
-
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
 
 /**
  * Created by jens on 15/07/17 in lagerverwaltung.
  */
 public class MainTest {
     @Test
-    public void testicle() throws Exception {
-        assertTrue(true);
+    public void initiateWarehouse() throws Exception {
+        Warehouse warehouse = new Warehouse();
+        Shelf shelf = new Shelf();
+        Compartment compartment = new Compartment();
+        final Item item = new Item();
+
+        shelf.initiateShelf(compartment);
+        warehouse.initiateShelves(shelf);
```

Comment:
how about you only tell the warehouse to `init` and it then calls its children's init methods so everything is initiated with default values